stl/inc/yvals_core.h

```diff
@@ -328,6 +328,7 @@
 // P1072R10 basic_string::resize_and_overwrite
 // P1132R7 out_ptr(), inout_ptr()
 // P1147R1 Printing volatile Pointers
+// P1169R4 static operator()

```

Comment:
(Looks back at `<functional>`) The code is guarded by `#ifdef _HAS_CXX17`; should this comment be in the ``_HAS_CXX17` section as well?

Reply:
Sure, done. Since Clang has unconditionally backported, we should ask MSVC and EDG to do that as well.

Reply:
I added a comment and pinged the responsible MSVC compiler developer. I suppose we need to send an email to the Edge team to request EDG support?